distribution/lib/Standard/Base/0.0.0-dev/src/Data/Filter_Condition.enso

```diff
@@ -178,9 +179,10 @@ type Filter_Condition
        Resolves a possibly auto-scoped value to a concrete value.
     resolve_auto_scoped : Any -> Any
     resolve_auto_scoped filter =
-        resolve filter:Filter_Condition = filter
         case filter of
-            _ : Function -> Panic.catch Type_Error (resolve filter) _->filter
```

Comment:
- This `catch` was annoying me a lot when debugging.
- I usually put breakpoint into `PanicException` to stop on an Enso error
- but this `catch` is raised tons of time and it doesn't mean an error
- hence I finally decided to implement #5226